docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+
+.. prompt:: bash [you@laptop:~$]
+
+	flocker-deploy 172.16.255.250 deployment-node1.yml fig.yml
+
+Visit http://172.16.255.250/ (or the IP of the first host that you are using). You will see the visit count displayed.
+
+Visit http://172.16.255.251/ (or the IP of the second host that you are using).
+You will see that the count persists because Flocker routes the traffic from either host named in the deployment file to the one that has the application.
+
+Run the following from within the :file:`/vagrant-flocker` folder to check that the Redis server container is running on the first host:
+
+.. prompt:: bash [you@laptop:~$]
+   
+   cd vagrant-flocker
+   vagrant ssh node1 -c "docker ps"
+
+You should see the Redis server container in the output from Docker.
+
+If you are running on AWS, manually SSH onto the first node and run :code:`docker ps` to see the same output.
+
+Step 4: Migrating a container to the second host
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The diagram below illustrates your current server-side Flocker setup:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :width: 60 %
+   :alt: In the server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+You'll need to download the last of the .yml files that we have provided:
+
+.. version-code-block:: console
+
+	you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node2.yml
+
+To move the container with the Redis server along with its data volume, use the deployment-node2.yml file:
+
+    .. literalinclude:: deployment-node2.yml
+       :language: yaml
+
+Run the following:
+
+.. prompt:: bash [you@laptop:~$]
+
+	flocker-deploy 172.16.255.250 deployment-node2.yml fig.yml
+
+The container on the Redis server and its volume have now both been moved to the second host.
+Flocker has maintained its link to the web application on the first host.
+
+Visit http://172.16.255.250/ (or the IP of the first host that you are using).
+You will see the visit count is still persisted.
+
+Visit http://172.16.255.251/ (or the IP of the second host that you are using).
+You will see that the count still persists, even though the container with the volume has moved between hosts.
+
+Run the following from within the vagrant-flocker folder to check that the Redis server container is running on the first host:
+
+.. prompt:: bash [you@laptop:~$]
+
+   cd vagrant-flocker
+   vagrant ssh node2 -c "docker ps"
+
+You should see the Redis server container in the output from Docker.
+
+If you are running on AWS, manually SSH onto the second node and run :code:`docker ps` to see the same output.
+
+Success!
+^^^^^^^^
+
+You have now set up your first Flocker cluster and moved a Docker container with its volume while persisting its link to a web app on another server.
```

Comment:
I think that this idea is missing through the guide. Even if it were mentioned here I think that this guide doesn't illustrate a core aim of an introduction: what problem I have which Flocker is solving.